conngen/conngenmodule.cpp

```diff
@@ -68,15 +67,14 @@ ConnectionGeneratorModule::init( SLIInterpreter* i )
   ConnectionGeneratorType.setdefaultaction( SLIInterpreter::datatypefunction );
 
   // Register the user functions of the connection generator interface
-  i->createcommand( "CGConnect_cg_i_i_D_l", &cgconnect_cg_i_i_D_lfunction );
-  i->createcommand( "CGConnect_cg_iV_iV_D_l", &cgconnect_cg_iV_iV_D_lfunction );
+  i->createcommand( "CGConnect_cg_g_g_D_l", &cgconnect_cg_g_g_D_lfunction );
   i->createcommand( "CGParse", &cgparse_sfunction );
   i->createcommand( "CGParseFile", &cgparsefile_sfunction );
   i->createcommand(
     "CGSelectImplementation", &cgselectimplementation_s_sfunction );
 
   // Register the low level functions of the connection generator interface
-  i->createcommand( "cgsetmask_cg_iV_iV", &cgsetmask_cg_iV_iVfunction );
+  i->createcommand( "cgsetmask", &cgsetmask_cg_g_gfunction );
```

Comment:
`cgsetmask` and some other functions are described in the documentation below as belonging to the low-level API. (a) Do they need to be part of the public API? (b) Should they get names beginning with `:` on the SLI level to mark them as internal?